tests/README.md

```diff
@@ -48,7 +48,7 @@ Run `./tests/codegen-tests.sh -u`. This updates all the codegen tests.
 Runtime tests will call the bpftrace executable. These are located in `tests/runtime` and are managed by a custom framework.
 
 * Run: `sudo make runtime-tests` inside your build directory or `sudo <builddir>/tests/runtime-tests.sh`
-* Use the `TEST_FILTER` environment variable (or the `--filter` arg when running `runtime_tests.sh`) to only run a subset of the tests e.g. `TEST_FILTER="uprobe.*" sudo make runtime-tests`
+* Use the `TEST_FILTER` environment variable (or the `--filter` arg when running `runtime-tests.sh`) to only run a subset of the tests e.g. `TEST_FILTER="uprobe.*" sudo make runtime-tests`
```

Comment:
This one was definitely wrong though! I think I must have meant to change the `make runtime-tests` in the line above instead